### ISSUE TITLE
The "PARENT" class constant is tokenized as T_PARENT

### ISSUE THREAD

**Issue by aik099** [squizlabs/PHP_CodeSniffer#267]
I had a following PHP code:

``` php

class UserType {
    const CAREGIVER  = 1;
    const PARENT = 2;
}

echo UserType::PARENT;
```

The sniff was listening for `T_PARENT` token only and last line of the code was matched. What I expected to match was only code that looks like `parent::` instead.

I had to do some pretty large workaround code to exclude false positives in https://github.com/aik099/CodingStandard/blob/master/CodingStandard/Sniffs/CodeAnalysis/WrongParentCallSniff.php . 

I remember that some `T_PARENT` token related fixes were committed recently, but I'm not aware if that one was there as well.
